README.md

```diff
@@ -1,102 +1,41 @@
+<p align="center">
 <picture>
   <source media="(prefers-color-scheme: dark)" srcset="website/static/img/logo_dark.svg">
   <img alt="Apache OpenDAL(incubating)" src="website/static/img/logo.svg" width="300px">
 </picture>
+</p>
 
-[![Build Status]][actions] [![chat]][discord]
```

Comment:
We could keep this.

Reply:
Not on `READMD.md`, we should show them in corresbonding components instead.